site/docs/.vuepress/configs/en.ts

```diff
@@ -358,9 +358,13 @@ export const localeEn: LocaleConfig<DefaultThemeLocaleData> = {
                 link: "/hosting/supabase.html",
               },
               {
-                text: "Cloudflare Workers",
+                text: "Cloudflare Workers (Node)",
```

Comment:
```suggestion
                text: "Cloudflare Workers (Node.js)",
```

Reply:
Perhaps we should use the old link to refer to the Deno guide, and then make a new page for Node.js? Deno tooling is far superior, so it makes sense to guide people there.